### ISSUE TITLE
Add suggestions to errors/warnings

### ISSUE THREAD

**Issue by zargot** [Big-Life-Lab/PHES-ODM-Validation#73]
Once we have a coersion library, we will probably want to update the warnings to give specific advice on what do to when a user receives a warning that their data can be coersed to an number. This made me wonder if we should have an addition error message section for "suggestions". Meaning, we would have "error", "warning", and "suggestion". We can always put a suggestion into an error or warning message, so this isn't really required. Just raising it as a thought or discussion point.

_Originally posted by @DougManuel in https://github.com/Big-Life-Lab/PHES-ODM-Validation/pull/64#pullrequestreview-1191376751_

**Comment by zargot** [comment]
Suggestions are a good idea. I would put them together with the error/warning to keep it simple and close together.

**Comment by zargot** [comment]
We would need to spec what the suggestions should be for each rule, @yulric.

**Comment by yulric** [comment]
> We would need to spec what the suggestions should be for each rule, @yulric.

Agreed re spec. I don't think this is a priority right and I'm not sure they would even look like without feedback from our partners. Lets wait on this.
